src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitorHelpers.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal class SymbolDisplayVisitorHelpers
+    {
+        internal static IMethodSymbol FindValidCloneMethod(ITypeSymbol containingType)
+        {
+            if (containingType.SpecialType == SpecialType.System_Object)
+            {
+                return null;
+            }
+
+            IMethodSymbol candidate = null;
+
+            // TO-DO: Change hard-coded '<Clone>$' to WellKnownMemberNames.CloneMethodName once it is made public
```

Comment:
>TO-DO: [](start = 15, length = 6)

We do not add TODO comments, please open an issue instead

Reply:
It is fine to add a reference to the issue here.

---
In reply to: [493951923](https://github.com/dotnet/roslyn/pull/47960#discussion_r493951923) [](ancestors = 493951923)

Reply:
I changed this back to internal so I don't think it is an issue anymore.